code/__HELPERS/roundend.dm

```diff
@@ -201,22 +201,20 @@
 		player_client.give_award(/datum/award/score/hardcore_random, human_mob, round(human_mob.hardcore_survival_score))
 
 
-/datum/controller/subsystem/ticker/proc/declare_completion()
+/datum/controller/subsystem/ticker/proc/declare_completion(was_forced = FALSE)
```

Comment:
why are we treating `was_forced` as a boolean when we directly compare it against defines that can be either `1` or `2`? i think this should default to a new define, something like `#define REGULAR_END_ROUND 0` to be super duper 100% clear because using boolean values for anything beyond a dichotomy makes me frown

Reply:
also i think these would be better as string defines instead of numerical defines for what we're doing here, but that's optional